src/instructlab/model/backends/vllm.py

```diff
@@ -91,15 +128,43 @@ def run_detached(self, http_client: httpx.Client | None = None) -> str:
 
     def shutdown(self):
         """Shutdown vLLM server"""
+
         # Needed when a temporary server is started
         if self.process is not None:
             shutdown_process(self.process, 20)
 
 
+def format_template(model_family: str, model_path: pathlib.Path) -> str:
+    template, eos_token, bos_token = get_model_template(model_family, model_path)
+
+    prefix = ""
+    if eos_token:
+        prefix = '{{% set eos_token = "{}" %}}\n'.format(eos_token)
+    if bos_token:
+        prefix = '{}{{% set bos_token = "{}" %}}\n'.format(prefix, bos_token)
+
+    return prefix + template
```

Comment:
Should we use a jinja lib to validate this template before sending it to vllm? Or build using a jinja lib maybe?

Reply:
(see above on that one, assuming I understood your comment correctly)